uwblueprint/sistering
# Add hotjar/GA integration for analytics

## Ticket link

Closes #298


## Implementation description
* Followed ingration steps in #298 and [this guide](https://medium.com/@romansorin/using-hotjar-and-google-analytics-with-next-js-to-track-behavior-a2283d659767) and [this guide](https://blog.saeloun.com/2022/02/17/how-to-integrate-react-app-with-google-analytics.html)


## Steps to test
1. Merge code into main and deploy to sistering-dev
2. Visit at least 2 different pages
3. Go to Hotjar/GA respectively, ensure that respective events shows up in analytics after a while

## Checklist
